docs/03-quickstart/05-ccd/03-aws-marketplace/01-byol.md

```diff
@@ -51,6 +50,11 @@ follow the following steps.
    add the `--profile your_profile` flag to the `aws` command below.
   
    ```shell showLineNumbers placeholders version=https://api.github.com/repos/strmprivacy/data-plane-helm-chart/releases/latest#name, installationId=Installation ID, clientId=Client ID of your installation, clientSecret=Client Secret of your installation
+
+   # create kubernetes namespace and set as default
```

Comment:
Not necessary, Helm creates the namespace if it doesn't exist.

Reply:
It doesn't, it got an error message that the namespace did not exist.